ucm2/conf.d/macaudio/iMac J456.conf

```diff
@@ -5,5 +5,10 @@ SectionUseCase."HiFi" {
 	# the same config. (That is until there will be support for the
 	# internal mic on the laptops.)
 	Comment "Default"
-	File "laptops1/HiFi.conf"
+	Define.JackCodec "cs42l83"
+	Define.PlaybackChannels "4"
+	Define.JackDevice "0"
+	Define.SpeakerDevice "1"
+	Define.SpeakerElem "Left Front"
```

Comment:
should be `"Left Woofer 1 Speaker Playback Volume"` if we use the same names in our dts as for the other devices.